TF-4073 Update variable set documentation (#301)
hashicorp/terraform-docs-common: website/docs/cloud-docs/workspaces/variables/managing-variables.mdx

1. Write an optional **Description** that tells other users about the purpose of the variable set and what it contains.

1. Choose a variable set scope:
   - **Apply globally:** Terraform Cloud will automatically apply this global variable set to all existing and future workspaces.

>    - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. Users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace) after creation.

Reviewer: It might be nice to have a short note here about the difference between applying the varset to a project vs. a workspace, maybe just something like:

> Note that applying a varset to a project will also apply that varset to all current and future workspaces in that project.

>    - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. This affects all current and future workspaces for any selected projects. Users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace) after creation.

Reviewer 2: ```suggestion
   - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. This affects all current and future workspaces for any selected projects. After creation, users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace).
```

>    - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. All current and future workspaces in any selected projects will access this variable set. Users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace) after creation.

Reviewer 3: ```suggestion
   - **Apply to specific projects and workspaces:** Use the text fields to search for and select workspaces and/or projects to apply this variable set to. This affects all current and future workspaces for any selected projects. Users can also [add this variable set to their workspaces](#apply-or-remove-variable-sets-from-inside-a-workspace) after creation.
```
